addons/hr_payroll/views/hr_benefit_views.xml

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record model="ir.actions.server" id="generate_benefit">
+        <field name="name">Generate Benefit</field>
+        <field name="model_id" ref="hr.model_hr_employee"/>
+        <field name="binding_model_id" ref="hr.model_hr_employee"/>
+        <field name="state">code</field>
+        <field name="code">
+if records:
+    records.generate_benefit()
+        </field>
+    </record>
+
+    <record id="hr_benefit_action" model="ir.actions.act_window">
+        <field name="name">Benefit</field>
+        <field name="res_model">hr.benefit</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">calendar,tree,form</field>
+    </record>
+
+    <record id="hr_benefit_calendar" model="ir.ui.view">
+        <field name="name">hr.benefit.calendar</field>
+        <field name="model">hr.benefit</field>
+        <field name="arch" type="xml">
+            <calendar string="Benefit"
+                date_start="date_start"
+                date_stop="date_stop"
+                mode="month"
+                quick_add="False"
+                color="benefit_type_id"
+                event_limit="5">
+                <field name="state" invisible="1"/>
+                <field name="employee_id" invisible="1"/>
+                <field name="display_warning" invisible="1"/>
+                <field name="name"/>
+                <templates>
+                    <t t-name="calendar-box">
+                        <t t-set="color" t-value="widget.getColor(event.color_index)"/>
+                        <div t-att-style="typeof color === 'string' ? ('background-color:'+color)+';' : ''" t-attf-class="#{record.is_highlighted &amp;&amp; record.is_highlighted.value ? 'o_event_hightlight' : ''} #{typeof color === 'number' ? 'o_calendar_color_'+color : ''}">
+                            <field name="name"/>
+                            <div class="fc-time"/>
+                            <div class="o_fields">
+                                <t t-if="record.display_warning">
+                                    <span class="fa fa-exclamation-triangle mr-2" aria-label="Conflict" title="error"/>
+                                </t>
+                                <t t-if="record.state == 'validated'">
+                                    <span class="fa fa-check-circle-o" aria-label="Validated" title="validated"/>
+                                    <t t-esc="' '"/>
```

Comment:
WTF?!

Reply:
I tried several other things to add a space such as `&nbsp;`.  None worked in the context of the calendar template.
I'll try to find a less hackinsh way and keep a nice layout.